mobile-app/src/lib/betterMock.ts

```diff
@@ -147,17 +147,27 @@ export const mockNotifications: allNotificationsT = {
 //   friend3: mockUsers['friend3'],
 // };
 export const mockFriendships: allFriendshipsT = {
+  friendship1: {
+    user1Id: "currentUserId123",
+    user2Id: "friend1",
+    friendsSince: new Date(),
+  },
   friendship2: {
     user1Id: "currentUserId123",
     user2Id: "friend2",
     friendsSince: new Date(),
   },
   friendship3: {
-    user1Id: "friend3",
-    user2Id: "currentUserId123",
+    user1Id: "currentUserId123",
+    user2Id: "friend3",
```

Comment:
I did this because we should have a mock friendship where the current user is user2. To make sure it works even when the currentUser isn't user1. So if you want to ensure they're in the proper order, maybe change "friend3" to "afriend3" everywhere or something?